src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -486,6 +501,16 @@ void DlgPrefSound::syncBuffersChanged(int index) {
     }
 }
 
+void DlgPrefSound::engineClockChanged(int index) {
+    if (index == 0) {
+        // "Soundcard Clock"
+        m_config.setForceNetworkClock(false);
```

Comment:
Does this take effect only after the Apply button is pressed?

Reply:
Yes, or the OK button.